pipeline/matching/core/__init__.py

```diff
@@ -1,3 +1,4 @@
 from pipeline.matching.core.constants import *
 from pipeline.matching.core.engine import *
 from pipeline.matching.core.generator import *
+from pipeline.matching.evaluation.analytics import *
```

Comment:
Delete this line, we should not cross-import packages in the `__init__.py` file.